internal/dag/builder_test.go

```diff
@@ -1664,9 +1664,7 @@ func TestDAGInsert(t *testing.T) {
 			),
 		},
 		"insert ingress w/ tls with different secure port": {
```

Comment:
Is this test testing anything anymore?

Reply:
No, and it was broken. Oops, thanks, removed.